Source/Core/Common/FileUtil.cpp

```diff
@@ -902,16 +900,15 @@ std::string CreateSysDirectoryPath()
 #endif
 
 #if defined(__APPLE__)
-  sysDir = GetBundleDirectory() + DIR_SEP + SYSDATA_DIR;
+  const std::string sysDir = GetBundleDirectory() + DIR_SEP SYSDATA_DIR DIR_SEP;
```

Comment:
You might want to change the name `sysDir` to `sys_dir` to make it compliant to [Dolphin's coding style](https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md#naming)